deployment/ccip/changeset/state.go

```diff
@@ -2143,3 +2145,8 @@ func validateLatestConfigOffRamp(offRamp offramp.OffRampInterface, cfg offramp.M
 	}
 	return nil
 }
+
+func getOfframpDynamicConfig(c deployment.AptosChain, ccipAddress aptos.AccountAddress) (module_offramp.DynamicConfig, error) {
+	offrampBind := ccip_offramp.Bind(ccipAddress, c.Client)
+	return offrampBind.Offramp().GetDynamicConfig(&aptosBind.CallOpts{})
+}
```

Comment:
The `changeset/state.go` is too generic. Could we have this function in `deployment/ccip/changeset/aptos_state.go`? Similar to what Solana is doing.